### DISCUSSION TITLE
Database connection

### DISCUSSION THREAD

**Discussion by ArtMin96** [sfx101/docker-stacks#127] [Q&A] [answered]
Thank you for such a wonderful product. I created a Laravel project and everything works fine, but I don't know how to connect to the database. Where can I create and connect to the project?

**Comment by nabad600** [comment, marked as answer]
Hi @ArtMin96 ,
Thanks for using DECK, please see the below-mentioned ticket, hope we are resolved your query. 
https://github.com/sfx101/deck/issues/59
